phpBB/includes/compatibility_globals.php

```diff
@@ -44,7 +44,6 @@
 // Grab global variables, re-cache if necessary
 /* @var $config phpbb\config\db */
 $config = $phpbb_container->get('config');
-set_config(null, null, null, $config);
```

Comment:
I think we actually need this somewhere. Otherwise set_config is not usable, although it is supposed to be still available.